html/semantics/embedded-content/the-iframe-element/iframe-loading-lazy-history-pushState.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<title>History state change for iframe loading='lazy' before it is loaded: history.pushState</title>
+<iframe data-src="support/blank.htm?foo" src="support/blank.htm?src" loading="lazy"></iframe>
+<script>
+const iframe = document.querySelector('iframe');
+// Should have no effect on lazy-loading
+iframe.contentWindow.history.pushState(null, "", iframe.dataset.src);
+</script>
+<!-- Loading testharness.js here is intentional. -->
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+setup({single_test: true});
+onload = () => {
```

Comment:
This doesn't work (reliably) because of https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-iframe-element:current-navigation-was-lazy-loaded-5

Reply:
I mean the `load` event might fire before the iframe is loaded, since a lazy-loading iframe doesn't block the `load` event. Therefore it's racy. But the `step_timeout` should address this sufficiently.